src/ASM/ASMs3D.h

```diff
@@ -199,9 +199,11 @@ class ASMs3D : public ASMstruct, public ASM3D
   //! \param[in] lIndex Local index of the boundary face
   //! \param nodes Array of node numbers
   //! \param basis Which basis to grab nodes for (0 for all)
-  //! \param local If \e true, return patch-local node numbers
-  virtual void getBoundaryNodes(int lIndex, IntVec& glbNodes, int basis,
-                                bool local) const;
+  //! \param thick Thickness of connection
+  //! \param local If true return patch-local node numbers
+  virtual void getBoundaryNodes(int lIndex, IntVec& glbNodes,
```

Comment:
Bruk 'nodes' her også istedet for 'glbNodes', så stemmer det med doxy, linje 200.